curriculum/challenges/english/16-the-odin-project/top-the-box-model/the-box-model-question-e.md

```diff
@@ -0,0 +1,38 @@
+---
+id: 64a669f77a7d00f97013ed0c
+title: The Box Model Question E
+challengeType: 15
+dashedName: the-box-model-question-e
+--- 
+# --description--
+
+Now that you understand the basic syntax of HTML and CSS, we’re going to get serious. The most important skills you need to master with CSS are positioning and layout. Changing fonts and colors is a crucial skill, but being able to put things exactly where you want them on a webpage is even more crucial. After all, how many webpages can you find where absolutely every element is just stacked one on top of another?
+
+Learning to position elements on a webpage is not that difficult once you understand just a few key concepts. Unfortunately, many learners race through learning HTML and CSS to get to JavaScript and end up missing these fundamental concepts. This leads to frustration, pain, (and funny gifs) because all the JavaScript skills in the world are meaningless if you can’t stick your elements on the page where you need them to be. So with that in mind, let’s get started.
+
+# --question--
+
+## --text--
+
+What is a crucial skill to master in CSS for positioning and layout of elements on a webpage?
```

Comment:
This seems reduntant.

The crucial skill to master for positioning and layout is learning how to position? Feels like saying the crucial skill for centring a div is learning how to centre a div